metadata.json

```diff
@@ -103,7 +103,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 3.0.0 < 5.0.0"
+      "version_requirement": ">= 4.5.2 < 5.0.0"
```

Comment:
shouldn't this be 4.6.1? As far as I know this is the first puppet4 version that is considered stable. Vox Pupuli bumped everything to 4.6.1. All based on [this statement](https://github.com/voxpupuli/community-triage/blob/master/modules/notes/2017-01-05.md#discussion)